### PR TITLE
Add dataset_classification to frontend security API

### PR DESCRIPTION
**Pull request by alyiwang**
Test with API get and post. 
Now can handle null value for dataset_classification, confidentiality, classification, retention_policy and geographic_affinity.

### CODE REVIEW

File: web/app/dao/DatasetsDAO.java [hunk 1]
```diff
@@ -2233,21 +2234,29 @@ public static DatasetSecurity getDatasetSecurityByDatasetUrn(String datasetUrn)
 
 	private static DatasetSecurity mapToDatasetSecurity(Map<String, Object> result) throws IOException {
 		ObjectMapper om = new ObjectMapper();
+		Map<String, Object> datasetClassification =
+				StringUtils.isBlank((String) result.get("dataset_classification")) ? null
```
Comment (mars-lan) [inline web/app/dao/DatasetsDAO.java]: Can probably replace most these ternary operators with getOrDefault in java 8.

Comment (alyiwang) [inline web/app/dao/DatasetsDAO.java]: Thanks for suggesting 'getOrDefault'. But in this case, the map does contain this key, it's just the value could be null or empty so the JSON mapping will throw error.

Comment (mars-lan) [inline web/app/dao/DatasetsDAO.java]: Argh never mind. Maybe just factor out the common code to a separate function then?

Comment (alyiwang) [inline web/app/dao/DatasetsDAO.java]: fixed, thanks

**Pull request merged**